main.go

```diff
@@ -473,6 +473,34 @@ func main() {
 			return err
 		}
 
+		//
+		// hc-launch
+		//
+		hcLaunchDescription := "tauri based CLI to run holochain apps in development mode"
+		hcLaunchRepositoryArgs := StandardRepositoryArgs("hc-launch", &hcLaunchDescription)
+
+		hcLaunch, err := github.NewRepository(ctx, "hc-launch", &hcLaunchRepositoryArgs, pulumi.Import(pulumi.ID("hc-launch")))
+		if err != nil {
+			return err
+		}
+		if err = RequireMainAsDefaultBranch(ctx, "hc-launch", hcLaunch); err != nil {
+		  return err
+		}
+		if err = StandardRepositoryAccess(ctx, "hc-launch", hcLaunch); err != nil {
+			return err
+		}
+		if err = AdditionalRepositoryAdmin(ctx, "hc-launch", "matthme", hcLaunch); err != nil {
```

Comment:
Not sure if we want to hard code accounts in here, or if you just should be added to the general admin group

Reply:
I have no strong opinion on that, as long as I can keep admin access on this particular repository.

Reply:
I allowed myself to temporarily resolve this conversation to get the PR merged and be able to continue with the next repo. I'm happy to address this with a follow-up PR if we decide we want to change it (it was already deployed anyway).
What are your thoughts about this @ThetaSinner ?